rcgen/examples/simple.rs

```diff
@@ -0,0 +1,38 @@
+#![allow(clippy::complexity, clippy::style, clippy::pedantic)]
+
+use rcgen::{date_time_ymd, Certificate, CertificateParams, DistinguishedName, DnType, SanType};
+use std::fs;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+	let mut params: CertificateParams = Default::default();
+	params.not_before = date_time_ymd(1975, 01, 01);
+	params.not_after = date_time_ymd(4096, 01, 01);
```

Comment:
Let's avoid octal constants:

```suggestion
	params.not_before = date_time_ymd(1975, 1, 1);
	params.not_after = date_time_ymd(4096, 1, 1);
```